### DISCUSSION TITLE
Webcam as a source.

### DISCUSSION THREAD

**Discussion by treche01** [AntonyCorbett/OnlyM#428] [Ideas]
I am working on setting up our KH system to be ready for returning to the KH.  Part of what I am setting up is to allow remote talks/comments to be displayed on the same TV as the JWL videos are using.  To do this I am using a video capture card.  It would be a nice feature to add a Webcam as a source. I know that may not be simple to do.  But I think it would help keep things simple for the user in this use scenario.

**Comment by scottys94** [comment]
If you're looking to display the webcam video on your TV screens, you can consider using OBS. You would add the webcam as a source in OBS and then project that to your TV screens. But if you have remote speakers or commenters, you can simply drag the Zoom window onto the TV's as well.

**Reply by treche01** [reply]
well I was hoping for something that would work smother then that. I did figure out something that will work. I can use VLC to Display the output of a capture card that would capture Zoom. then it would be just a click.

**Reply by scottys94** [reply]
OBS also has a virtual camera that can be used in Zoom. Experiment with your options and find what works best.
